mage_ai/io/qdrant.py

```diff
@@ -0,0 +1,173 @@
+from typing import List
+
+from pandas import DataFrame
+from qdrant_client import QdrantClient
+from qdrant_client.http import models
+from sentence_transformers import SentenceTransformer
+
+from mage_ai.io.base import BaseIO
+from mage_ai.io.config import BaseConfigLoader, ConfigKey
+
+DEFAULT_EMBEDDING_MODEL = 'all-MiniLM-L6-v2'
+
+
+class Qdrant(BaseIO):
+    def __init__(
+            self,
+            collection: str,
+            path: str = None,
+            verbose: bool = True,
+            **kwargs,) -> None:
+        """
+        Initializes connection to qdrant db.
+        """
+        super().__init__(verbose=verbose)
+        self.collection = collection
+        self.path = path
+        self.open()
+
+    @classmethod
+    def with_config(cls, config: BaseConfigLoader) -> 'Qdrant':
+        return cls(
+            collection=config[ConfigKey.QDRANT_COLLECTION],
+            path=config[ConfigKey.QDRANT_PATH],
+        )
+
+    def create_collection(
+            self,
+            vector_size: int,
+            distance: models.Distance = None,
+            collection_name: str = None):
+        """
+        Create collection in qdrant db.
+        Args:
+            vector_size (int): dimension size of the vector.
+            distance (models.Distance): distance metric to use.
+            collection_name (str): name of the collection.
+            Defaults to the name defined in io_config.yaml.
+        Returns:
+            collection created.
+        """
+        collection_name = collection_name or self.collection
+        distance = distance or models.Distance.COSINE if distance is None else distance
```

Comment:
should this just be `distance = models.Distance.COSINE if distance is None else distance`

Reply:
Actually updated it to `distance or models.Distance.COSINE` to be simplify